Cargo.toml

```diff
@@ -13,20 +13,16 @@ categories = ["no-std", "embedded"]
 exclude = [ "demo/*" ]
 
 [dependencies]
+fixed-hash = { version = "0.3.0-beta.0", default-features = false }
```

Comment:
I think you can use `version = "0.3.0-beta"` here. Saves some time if there are other pre-releases.

Reply:
ah that's good to know! thanks. :)

Reply:
done